pages/_app.tsx

```diff
@@ -1,4 +1,5 @@
-import '@/styles/globals.css'
+// import '@/styles/globals.css'
```

Comment:
Remove comments, this comment is neither a description nor describing a functionality, for these reason should be removed.